apps/costcontrol/test/unit/widget_startup_test.js

```diff
@@ -461,4 +453,82 @@ suite('Widget Startup Modes Test Suite >', function() {
     AirplaneModeHelper._status = 'disabled';
     Widget.init();
   });
+
+  test('DSDS normal start-up after sim switch from dataUseOnly to prepaid mode',
```

Comment:
Please, reword this: `DSDS start-up after SIM switching: from data usage only SIM to prepaid SIM`.